### ISSUE TITLE
Unrecognized resource type: ethereum_network - Transactions ETH

### ISSUE THREAD

**Issue by stevelyzo** [coinbase/coinbase-php#144]
I have a PHP application
Do I use the Coinbase API.
Now I want to see the transaction history of an address when I transfer ETH money to my wallet.
It reports an error.

#inclue 
```
use Coinbase\Wallet\Value\Money;
use Coinbase\Wallet\Resource\Account;
use Coinbase\Wallet\Resource\Address;
```

#function 
```
$configuration = Configuration::apiKey($coin_Key, $coin_Secret);
$client = CoinClient::create($configuration);
$accounts = $client->getAccounts();
$accountId = $accounts->getFirstId(); 
$account = $client->getAccount($accountId);
$addresses = $client->getAccountAddresses($account);
$f_add = $addresses->getFirstId();
$address = $client->getAccountAddress($account, $f_add);

$transactions = $client->getAddressTransactions($address);

dd($transactions);

```

#Error message 

Unrecognized resource type: ethereum_network

**Comment by Doudlidou** [comment]
Hi, same with Litecoins...

**Comment by shked0wn** [comment]
Fixed in #138, you can use my fork in order to work

**Comment by Doudlidou** [comment]
Hi btw thanks for your hard work in fixing their API, but i am using your fork and i am seeing this issue, I can create addrress for each coins but i can only get the balance for BTC....

**Comment by shked0wn** [comment]
@Doudlidou which endpoint are you using ? Can you copy/paste the result ? I will take a look ;)

**Comment by Doudlidou** [comment]
Hi My bad, did something wrong, your fork works fine. Just needed to add LTC, ETH in the currencyCode file.

Thanks again

**Comment by shked0wn** [comment]
@Doudlidou Ok great, you are welcome :)

**Comment by PriyanshDwivedi** [comment]
Hi @shked0wn , I am trying to send ETH using your fork changes and facing the below error. Would be great if you can advise as I have tried everything from my end.

My Code:
```
$account = $client->getAccount($accountId);

  $transaction = Transaction::send([
      'to' => new EthereumAddress($_GET['to']),
      'amount' => Money::eth($_GET['amount'])
  ]);
$client->createAccountTransaction($account, $transaction);
```

EthereumAddress.php kept in /coinbase/coinbase/src/Resource folder
```
<?php

namespace Coinbase\Wallet\Resource;

use Coinbase\Wallet\Enum\ResourceType;

class EthereumAddress extends Resource
{
    private $address;

    public function __construct($address)
    {
        parent::__construct(ResourceType::ETHEREUM_ADDRESS);

        $this->address = $address;
    }

    public function getAddress()
    {
        return $this->address;
    }
}
```

Error:
```
response:\n{"errors":[{"id":"param_required","message":"Required parameter missing: to"}]
```

**Comment by shked0wn** [comment]
Hello @PriyanshDwivedi, where did you find this `EthereumAddress` class ?  I can't find it.

Have you tried to do this :
  
new Address($_GET['to']) instead of new EthereumAddress($_GET['to']) ?

**Comment by PriyanshDwivedi** [comment]
I modified their source code and added few missing files to facilitate sending Ethereum and generating Ethereum addresses and got it working fine last night.

**Comment by sds** [comment]
Fixed in #138.
